### PR TITLE
Update foreman-tasks to 0.10.0 (DEB)

### PR DESCRIPTION
**Pull request by iNecas**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.15
* [ ] 1.14
* [ ] 1.13

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

File: plugins/ruby-foreman-tasks/debian/gem.list [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # list of gem urls to download via Jenkins, space separated
-GEMS="https://rubygems.org/downloads/foreman-tasks-0.9.3.gem"
-GEMS="$GEMS https://rubygems.org/downloads/foreman-tasks-core-0.1.3.gem"
+GEMS="https://rubygems.org/downloads/foreman-tasks-0.10.0.gem"
+GEMS="$GEMS https://rubygems.org/downloads/foreman-tasks-core-0.2.0.gem"
 GEMS="$GEMS https://rubygems.org/downloads/daemons-1.2.4.gem"
```
Comment (mmoll) [inline plugins/ruby-foreman-tasks/debian/gem.list]: this seems to be in Foreman core with recent nightlies now and needs to get removed here

**Comment by mmoll** [comment]
hm, I guess the dynflow plugin is <1.16 only now, so the packaging needs to change and there will be a split in 1.15 and nightly PRs?

**Comment by iNecas** [comment]
You mean dynflow needs to get out of the gem, right? This is ok, as we are not targeting 0.10.0 for older releases (we will release 0.9.x if needed in 1.15)

File: plugins/ruby-foreman-tasks/debian/gem.list (hunk 1 above)
Comment (iNecas) [inline plugins/ruby-foreman-tasks/debian/gem.list]: Good point, updated.

**Comment by mmoll** [comment]
[test]

**Comment by mmoll** [comment]
merged, díky @iNecas!

**Pull request merged**